### DISCUSSION TITLE
[auto] Change CMD in Dockerfile to use array format instead of string

### DISCUSSION THREAD

**Discussion by DevilaN** [AbdBarho/stable-diffusion-webui-docker#511] [Ideas]
As for now string is used, which forces docker to use `/bin/sh -c` to run command provided in string.
Unfortunately this ends up with `sh` process being PID=1 and getting all signals.
python is spawned as child process of `sh` and does not receive signals.

When ctrl+C is sent then `sh` process dies and makes python an orphaned process which does not close properly. Thats why it does not react immediately to stop signal and ends up with fancy exit code.

Is there any reason to use string CMD format?

**Comment by AbdBarho** [comment]
> Is there any reason to use string CMD format?

Not really, it is just much easier when you have to type 20 arguments, feel free to create PR, maybe we would not need to change the signal after all.
